<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QLabel" name="label_validation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Invalid Credentials&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>196</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>Username: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_username"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_password"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_logout">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCustomer">
    <property name="title">
     <string>Customer</string>
    </property>
    <addaction name="actionSales_Pitch"/>
    <addaction name="actionConcept_of_operations"/>
    <addaction name="actionService_options"/>
    <addaction name="actionMaintenance_plan"/>
    <addaction name="actionGuarantee_Policy"/>
    <addaction name="actionCustomer_Testimonials"/>
    <addaction name="actionRequest_Pamphlet"/>
    <addaction name="actionOrder_Products"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCustomer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSales_Pitch">
   <property name="text">
    <string>Sales Pitch</string>
   </property>
  </action>
  <action name="actionConcept_of_operations">
   <property name="text">
    <string>Concept of Operations</string>
   </property>
  </action>
  <action name="actionService_options">
   <property name="text">
    <string>Service Options</string>
   </property>
  </action>
  <action name="actionMaintenance_plan">
   <property name="text">
    <string>Maintenance Plan</string>
   </property>
  </action>
  <action name="actionGuarantee_Policy">
   <property name="text">
    <string>Guarantee Policy</string>
   </property>
  </action>
  <action name="actionCustomer_Testimonials">
   <property name="text">
    <string>Customer Testimonials</string>
   </property>
  </action>
  <action name="actionRequest_Pamphlet">
   <property name="text">
    <string>Request Pamphlet</string>
   </property>
  </action>
  <action name="actionOrder_Products">
   <property name="text">
    <string>Order Products</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
